release/artifacts.yaml

```diff
@@ -56,15 +56,19 @@
   type: jar-scala
 - target: //ledger-api/grpc-definitions:ledger-api-protos-tarball
   type: targz
+  mavenUpload: True
```

Comment:
How do we decide what is uploaded to Maven and what isn’t?

Reply:
Currently only the Java and Scala bindings and code generator tools along with any dependencies are to be released (decided by @bitonic).